app/products/calls/utils.ts

```diff
@@ -14,6 +15,8 @@ import type PostModel from '@typings/database/models/servers/post';
 import type {IntlShape} from 'react-intl';
 import type {RTCIceServer} from 'react-native-webrtc';
 
+const callsMessageRegex = /^\u200b.* is inviting you to a call$/;
```

Comment:
we should add a note here not to forget when the plugin starts to support localization

Reply:
This regex won't need to be changed when we support localization -- when we support localization it will be handled by the sub_type check. The regex will still be needed to match older messages from pre-0.21.0 calls (or pre-change MM server and pre-change proxy servers). When we finally remove that section of the code below, this can be removed (it will show up as dead-code and go will tell us).